libraries/provision/install_sync_gateway.py

```diff
@@ -134,7 +134,7 @@ def install_sync_gateway(cluster_config, sync_gateway_config, sg_ce=False):
 
     if is_xattrs_enabled(cluster_config):
         playbook_vars["autoimport"] = '"import_docs": "continuous",'
-        playbook_vars["xattrs"] = '"enable_extended_attributes": true'
+        playbook_vars["xattrs"] = '"enable_shared_bucket_access": true,'
```

Comment:
comma is needed. In the json, {{ xattrs }} gets replaced with "enable_shared_bucket_access": true, . Without comma, it would be an invalid json.